packages/minuteman/buildinfo.json

```diff
@@ -4,7 +4,7 @@
     "minuteman": {
       "kind": "git",
       "git": "https://github.com/dcos/minuteman.git",
-      "ref": "1c45dab611b13cb3a491a2b41f2dc7d36ffde7b2",
+      "ref": "5fff0caf44d15f734cfb961338b04aec1dc3424f",
       "ref_origin": "dcos/1.8"
```

Comment:
I don't see the equivalent of this commit anywhere on the `dcos/1.8` branch for `navstar`:
```
commit 4079543987fd7b1c0b03b97110e2a97a883b8dcd
Merge: 68c634f af5a550
Author: Deepak Goel <dgoel@mesosphere.com>
Date:   Thu Mar 23 01:21:52 2017 -0700

    Merge pull request #58 from dcos/fix-dcos-oss-804
    
    log message to include pid, module, function, line number
```

Amy I missing something?